src/Components/CollectionsRail/CollectionsRail.tsx

```diff
@@ -0,0 +1,61 @@
+import { Box, Flex, Sans, Spacer } from "@artsy/palette"
+import { CollectionsRail_collections } from "__generated__/CollectionsRail_collections.graphql"
+import React, { SFC } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
+
+import { pMedia } from "Components/Helpers"
+import { CollectionEntityFragmentContainer as CollectionEntity } from "./Collection"
+
+interface CollectionRailsProps {
+  collections: CollectionsRail_collections
+}
+
+const RailsWrapper = styled(Flex)`
+  width: 100%;
+  max-width: 1250px;
+  margin: 0 auto;
+  flex-direction: column;
+  ${pMedia.xl`
```

Comment:
We are trying to deprecate `pMedia` in favor of Palette's `media` wherever possible, though I'm guessing you are using this to match to further reading or another component that still uses it... probably worth adding a TODO to remove later.

Reply:
Yes, that's correct. I'll add a TODO. It also might be good to add as a task to the ticket for closing the test in the event this version wins.